tests/test_quickstart.py

```diff
@@ -29,7 +29,7 @@ def test_quickstart():
         for center in centers:
             center = jnp.array(center)
             try:
-                spectrum, morph = init.adaptive_morphology(obs, center, min_corr=0.99)
+                spectrum, morph = init.make_bbox(obs, center, min_corr=0.99)
```

Comment:
This must be incorrect. That method only returns a box, but we need `spectrum, morph = init.adaptive_morphology(obs, center, min_corr=0.99)`. Please change back.

Reply:
Right, I agree! Please check the related comment bellow

Reply:
OK, we're both wrong. The drop-in replacement for this method is `init.from_gaussian_moments`. Please change that

Reply:
Right, this is fixed now :-)